xwiki-pro-macros-ui/src/main/resources/XWiki/Macros/Team.xml

```diff
@@ -852,6 +852,15 @@
   #end
   {{html wiki=false clean=false}}
   &lt;div class="xwikiteam #if(!$showUsernames)xwikiteam-usernames-hidden#end"&gt;
+  ## Starting with XWiki version 13.10.4 and 14.1-rc-1, img tags have a height: auto set and this breaks the macro fixed
+  ## size feature. For this, a specific size is added
+  ## Image not to scale on mobile XWIKI-19432
```

Comment:
A suggestion:
`Starting with XWiki 13.10.4 and 14.1, img tags have a 'height: auto' style set, see XWIKI-19432: Image not to scale on mobile. Since this breaks the macro fixed size feature, a specific size is added.`